src/content/h1/tools/hek-plus/readme.md

```diff
@@ -0,0 +1,34 @@
+**Halo Editing Kit Plus (HEK+)** is a tool for extracting tags from Halo Custom
+Edition maps, as well as "protecting" maps to prevent tag extraction.
+
+```.alert danger
+**HEK+ should not be used!** It introduces a number of issues to the tags it
+extracts.
```

Comment:
Soft wrapping would be preferable for me over hard wrapping at 80 chars, makes paragraphs a bit easier to edit later without worrying about lines.